projects/github-actions/repo-gardening/src/tasks/add-labels/index.js

```diff
@@ -137,18 +137,26 @@ async function getLabelsToAdd( octokit, owner, repo, number, isDraft, isRevert )
 			}
 		}
 
-		// The SSO feature nows lives in both a package and a Jetpack module.
+		// The SSO feature now lives in both a package and a Jetpack module.
 		const sso = file.match( /^projects\/packages\/connection\/src\/sso\// );
 		if ( sso !== null ) {
 			keywords.add( '[Feature] SSO' );
 		}
 
-		// The Google Analytics feature nows lives in both a package and a Jetpack module.
+		// The Google Analytics feature now lives in both a package and a Jetpack module.
 		const googleAnalytics = file.match( /^projects\/packages\/google-analytics\// );
 		if ( googleAnalytics !== null ) {
 			keywords.add( '[Feature] Google Analytics' );
 		}
 
+		// The Publicize feature now lives in a package, a Jetpack module, and a js package.
+		const publicize = file.match(
+			/^projects\/(packages\/publicize)|(js-packages\/publicize-components)\//
```

Comment:
Did you mean to do like this?
```suggestion
			/^projects\/(packages\/publicize|js-packages\/publicize-components)\//
```

Reply:
Yes I did. My IDE failed me on that one I think.